config/deploy/production.rb

```diff
@@ -2,5 +2,7 @@
 
 set :stage, :production
 set :rails_env, 'production'
+set :linked_dirs, %w[log]
```

Comment:
Does this need to be in the environment-specific files, or can it go in a single line in the deploy.rb?

Reply:
`config/deploy.rb` is the file